pkg/disk/bdf.go

```diff
@@ -594,11 +594,17 @@ func NewDeviceDriver(volumeId, blockDevice, deviceNumber string, _type MachineTy
 			}
 		}
 	} else {
-		output, err := utils.CommandOnNode("xdragon-bdf", "--nvme", "-id=%s", volumeId).CombinedOutput()
-		if err != nil {
-			return nil, fmt.Errorf("Failed to excute bdf command: %s, err: %v", volumeId, err)
+		for _, pciDriver := range []string{"--nvme", "--blk"} {
+			output, err := utils.CommandOnNode("xdragon-bdf", pciDriver, "-id=%s", volumeId).CombinedOutput()
+			if err != nil {
+				return nil, fmt.Errorf("Failed to excute bdf command: %s, err: %v", volumeId, err)
+			}
+			bdf := strings.TrimSpace(string(output))
+			if bdf != "" {
+				d.deviceNumber = string(output)
```

Comment:
```suggestion
				d.deviceNumber = bdf
```
I assume you mean this?

Reply:
yeah， typo